entity/enemy.py

```diff
@@ -3,27 +3,81 @@
 import arcade
 
 from constants import TILE_SIZE
+from utils import Vector, center_of_tile
 
+class pathcolors:
+    pathcoloridx = 0
+    pathcolorlist = [
+        (97, 82, 103),
+        (34, 61, 40),
+        (57, 48, 51),
+        (230, 211, 179),
+        (65, 115, 76),
+        (120, 101, 128)]
+
+    @classmethod
+    def get_color(cls):
+        color = cls.pathcolorlist[cls.pathcoloridx]
+        cls.pathcoloridx += 1
```

Comment:
i don't think we should reuse the same colors, this might be confusing to the player. if we need more than 6 guards we should just add more colors.

Reply:
Ed originally gave me 8 colors options, but I threw two of them out because they were hard to see against the floor.  Your suggestion makes sense. If I remove the `%=` and just let it error, it should naturally come up when testing such levels and be easy to fix.  Not sure how many we need or if there is a method than hardcoding our colors.  I think we'll likely need more than 6 guards.